snippets/variant-list-row.liquid

```diff
@@ -299,3 +364,17 @@
     {% # theme-check-enable %}
   </td>
 </tr>
+
+<volume-pricing class="hidden" id="Volume-{{ variant.id }}">
```

Comment:
To get the volume pricing, I had to create a hidden volume pricing element to access it

But maybe we can look into an API later